test/Tester/HostedTestClusterBase.cs

```diff
@@ -14,7 +14,7 @@ public abstract class HostedTestClusterEnsureDefaultStarted : OrleansTestingBase
     {
         protected TestCluster HostedCluster { get; private set; }
 
-        public HostedTestClusterEnsureDefaultStarted(DefaultClusterFixture fixture)
+        public HostedTestClusterEnsureDefaultStarted(BaseTestClusterFixture fixture)
```

Comment:
Ok. Sorry, there a lot of implicits in how unit tests are run. Never used xunit before